backend/src/app/auth.gleam

```diff
@@ -0,0 +1,34 @@
+import birl
+import gleam/json
+import gleam/result
+import gwt
+import simplifile
+import wisp
+
+pub fn generate_jwt(userid: Int, username: String) {
+  let builder =
+    gwt.new()
+    |> gwt.set_issuer("hardwareby.us")
+    |> gwt.set_subject(username)
+    |> gwt.set_audience("hardwareby.us")
+    |> gwt.set_issued_at(birl.to_unix(birl.now()))
+    |> gwt.set_expiration(birl.to_unix(birl.now()) + 600)
```

Comment:
Is this set to expire in 600 seconds after we issue it?

I don't think we intend to use cookies/local storage to store this, so we're expecting the user to sign in every time. I think we might want the expiration to last long enough so that they get logged out while they are still navigating it.

Reply:
If we extend the expiration time, we could just store the JWTs in a table in the database with the userid as a foreign key.

Reply:
JWTs are inherently a "fire and forget" auth system; the relevant user data is stored inside the token payload (it's just JSON data) and the signature ensures validity of the token (by checking for a match on the server). 

As such, you'll usually keep em in local storage (i think cookies are less secure? not sure) and send on every request.